app/src/main/java/com/google/droidjump/GameConstants.java

```diff
@@ -27,6 +27,11 @@
     int SLEEP_TIME = 15; // Sleep time in gameView (ms).
     int GAME_LEVELS_COUNT = 100; // Count of game levels.
 
-    // String for identifying current level from SharedPreferences.
-    String GAME_VIEW_LEVEL_STRING = "java.com.google.droidjump.GameView.Level";
+    String GAME_LEVEL_HEADER = "Lvl."; // Used for displaying the current level in GameView.
```

Comment:
Should it be somewhere where other visible strings are stored? In resources, string.xml file?

Reply:
Not sure about it. @redls What do you think?

Reply:
I would leave it here for now.